README.md

```diff
@@ -0,0 +1,177 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 OOP School Library <a name="about-project"></a>
+
+**OOP School Library** is an app used to manage a school library by providing records on school books, teachers and students and the flow of books to and from the library. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add new students**
+- **Add new teachers**
+- **Add new books**
+- **Save records of books**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Not live yet
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps. 
+
+### Prerequisites
+In order to run this project you need:
+
+- Install ruby using [link](https://www.ruby-lang.org/en/documentation/installation/#apt)
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  mkdir folder_name
+  cd folder_name
+  git clone git@github.com:microvese-projects/OOP_school_library.git
+```
+-
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+```
+-
+
+### Usage
+
+To run the project, start the irb interactive platform:
+```sh
+  irb
+```
+-
+
+### Run tests
+
+No tests available.
+
+### Deployment
+
+Not deployed 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Benson Njuguna Kamau**
+
+- GitHub: [@bennyjoez](https://github.com/bennyjoez)
+- Twitter: [@bennyjoezz](https://twitter.com/bennyjoezz)
+- LinkedIn: [Benson Njuguna Kamau](https://www.linkedin.com/in/bennyjoez/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Webpage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/microvese-projects/OOP_school_library.git/issues).
```

Comment:
- [ ] I wanted to kindly bring to your attention a broken link in the README file of our project/repository. The `Open Issue` link, unfortunately, seems to be leading to an error page. Please fix this :point_down: 
![Screenshot from 2023-06-26 16-58-55](https://github.com/microvese-projects/OOP_school_library/assets/88894525/9a6bf2d7-0e42-4767-ad2a-57be7aa518a9)